requirements/requirements.waf.txt

```diff
@@ -1 +1 @@
-metlo
\ No newline at end of file
+metlo~=0.1.5
```

Comment:
@PawelPeczek-Roboflow this change is causing my build to fail (onnx.cpu.slim image)
```17.60 ERROR: Ignored the following versions that require a different python version: 0.23.0 Requires-Python >=3.10; 0.23.0rc0 Requires-Python >=3.10; 0.23.0rc2 Requires-Python >=3.10; 0.23.1 Requires-Python >=3.10; 0.23.2 Requires-Python >=3.10; 0.23.2rc1 Requires-Python >=3.10; 0.25.0rc0 Requires-Python >=3.10; 0.25.0rc1 Requires-Python >=3.10; 1.14.0 Requires-Python >=3.10; 1.14.0rc1 Requires-Python >=3.10; 1.14.0rc2 Requires-Python >=3.10; 1.14.1 Requires-Python >=3.10; 2.1.0 Requires-Python >=3.10; 2.1.0rc1 Requires-Python >=3.10; 2.1.1 Requires-Python >=3.10; 2.1.2 Requires-Python >=3.10; 2.1.3 Requires-Python >=3.10; 2024.9.20 Requires-Python >=3.10; 3.10.0rc1 Requires-Python >=3.10; 3.3 Requires-Python >=3.10; 3.3rc0 Requires-Python >=3.10; 3.4 Requires-Python >=3.10; 3.4.1 Requires-Python >=3.10; 3.4.2 Requires-Python >=3.10; 3.4rc0 Requires-Python >=3.10
17.60 ERROR: Could not find a version that satisfies the requirement metlo-python-agent-bindings-common==0.0.2 (from metlo) (from versions: none)
17.61 ERROR: No matching distribution found for metlo-python-agent-bindings-common==0.0.2```